app/src/main/res/layout/interation_feedback_card_item.xml

```diff
@@ -0,0 +1,25 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+  <data>
+    <variable
+      name="htmlContent"
+      type="String"/>
+  </data>
+  <RelativeLayout
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="24dp"
```

Comment:
Prefer ``marginStart`` and ``marginEnd`` rather than left/right to ensure that this layout works properly in RTL.

Reply:
Ditto elsewhere in this PR.

Reply:
corrected.